lib/ionic_notification/sent_notification.rb

```diff
@@ -0,0 +1,30 @@
+module IonicNotification
+  class SentNotification
+    attr_accessor :tokens, :title, :message, :android_payload,
+      :ios_payload, :production, :result, :message_id, :sent_at
+
+    def initialize(options = {})
+      @tokens = options[:tokens]
+      @title = options[:title]
+      @message = options[:message]
+      @android_payload = options[:android_payload]
+      @ios_payload = options[:ios_payload]
+      @production = options[:production]
+      @result = options[:result]
+      @message_id = options[:message_id]
+      @sent_at = Time.now
+      @scheduled = init_scheduled(options[:scheduled])
+    end
+
+    def status
+      service = StatusService.new @message_id
+      service.check_status!
+    end
+
+    private
+
+    def init_scheduled(timestamp)
+      return DateTime.strptime(timestamp.to_s,'%s') if timestamp
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.